gum/backend-libdwarf/gumsymbolutil-libdwarf.c

```diff
@@ -344,6 +356,58 @@ gum_find_function (const gchar * name)
   return address;
 }
 
+static gboolean
+gum_find_nearest_symbol_by_address (const gpointer address,
+                                    GumNearestSymbolDetails * nearest)
+{
+  GHashTable * table;
+  gchar * name;
+  GList * keys;
+  GList * current;
+  gpointer current_address;
+  gpointer closest_address = NULL;
```

Comment:
```suggestion
  gpointer current_address, closest_address;
```

Reply:
We need to NULL initialize closest_address. But I have merged the lines.